<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizadorMainWindow</class>
 <widget class="QMainWindow" name="VisualizadorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>585</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VisualizadorMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="mplwindow" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplvl"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="botaoArquivo">
         <property name="text">
          <string>Carregar arquivo</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Arquivo carregado:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="textoInfo1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Colorir partículas por:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="mplfigs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>200</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Escala de cores</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QRadioButton" name="radioButton_automatica">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>1013</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Automática</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_manual">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>1013</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>Manual</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="minimo">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>90</y>
            <width>91</width>
            <height>27</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
         </widget>
         <widget class="QLineEdit" name="maximo">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>130</y>
            <width>91</width>
            <height>27</height>
           </rect>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>31</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>31</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
         <widget class="QPushButton" name="botaoAplicar">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>170</y>
            <width>99</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Aplicar</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
